### PR TITLE
Load words.json file

### PR DESCRIPTION
**Pull request by hcrowell**
Load words.json file

### CODE REVIEW

File: plugins/plugin_word_respond.py [hunk 1]
```diff
@@ -13,7 +14,8 @@ def __init__(self, slack_client=None, plugin_config=None):
             slack_client=slack_client,
             plugin_config=plugin_config
         )
-        self.words = self.load_word_list("words.txt")
+        self.words = self.load_json("words.json")
+        print(self.words)
```
Comment (jkvoorhis) [inline plugins/plugin_word_respond.py]: rogue print statement

File: plugins/plugin_word_respond.py [hunk 2]
```diff
@@ -32,8 +34,20 @@ def process_message(self, data):
...
 
     def load_word_list(self, filepath):
         with open(filepath) as f:
             words = f.read().split()
             return words
+
+    def load_json(self, filepath):
+        with open(filepath) as word_file:
+            data = json.load(word_file)
+            return data
```
Comment (jkvoorhis) [inline plugins/plugin_word_respond.py]: python files have a newline at the end of their files according to PEP8

**Review by jkvoorhis — CHANGES_REQUESTED** [review]
two small things and then it's good to go!

File: plugins/plugin_word_respond.py (hunk 1 above)
Comment (hcrowell) [reply plugins/plugin_word_respond.py]: oops! good catch (i love my print debug statements)

**Review by jkvoorhis — APPROVED** [review]

**Pull request merged**